.gitignore

```diff
@@ -0,0 +1,8 @@
+/vendor/*
+*.tmp
+*.exe
+routers/commentsRouter__*
+.idea
+data
+.zip
+.rar
```

Comment:
Add `API` to this file to avoid to upload the generated binary.

Reply:
An .exe~ came by mistake